services/web-server/src/monitor.js

```diff
@@ -46,4 +46,21 @@ monitorManager.register({
   },
 });
 
+monitorManager.register({
+  name: 'requestReceived',
+  title: 'Request Received',
+  type: 'request-received',
+  version: 1,
+  level: 'info',
+  description: 'A GraphQL request has been received.',
+  fields: {
+    operationName: `
+      The name of the graphql query performed. If the operation is anonymous
+      (i.e., the operation is query { ... } instead of query NamedQuery { ... })
+      , then operationName is null.`,
```

Comment:
I don't see `query` described here -- does that contain the query content?

Reply:
Added.